app/src/features/surveys/view/survey-animals/TelemetryDeviceForm.tsx

```diff
@@ -7,58 +7,113 @@ import { Form, useFormikContext } from 'formik';
 import useDataLoader from 'hooks/useDataLoader';
 import { useTelemetryApi } from 'hooks/useTelemetryApi';
 import moment from 'moment';
-import { useEffect, useState } from 'react';
-import { IAnimalTelemetryDevice } from './animal';
+import { Fragment, useEffect, useState } from 'react';
+import { IAnimalDeployment, IAnimalTelemetryDevice } from './device';
 
-const TelemetryDeviceForm = () => {
-  const { values, setStatus } = useFormikContext<IAnimalTelemetryDevice>();
+export type TelemetryDeviceFormMode = 'add' | 'edit';
```

Comment:
Enums are a little nicer to work with if you use `enum`:

```javascript
export enum TELEMETRY_DEVICE_FORM_MODE {
  ADD = 'add',
  EDIT = 'edit'
}
```

Then you can refer to the add more as `TELEMETRY_DEVICE_FORM_MODE.ADD`